Source/relay/TourGuide/Paths/Avatar of Shadows over Loathing.ash

```diff
@@ -0,0 +1,25 @@
+RegisterTaskGenerationFunction("PathShadowsOverLoathingGenerateTasks");
+void PathShadowsOverLoathingGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [int] optional_task_entries, ChecklistEntry [int] future_task_entries)
+{
+    if (lookupSkill("Free-For-All").have_skill()) {
+        if ($effect[Everything Looks Red].have_effect() > 0) return;
```

Comment:
we probably should also have red rocket stuff not appear in AoSOL as well, similar to the YR stuff noted here

Reply:
Pushed a commit that removes the individual fireworks messages corresponding to each class. I decided to filter them when you're each class without checking whether you actually have the skill – even if you don't have it now, you'll probably get it by the time the ELR/B/Y cooldown finishes with the way leveling is now. Seem right?

Reply:
yep that sound right to me!